src/components/selectField.js

```diff
@@ -0,0 +1,16 @@
+//https://gist.github.com/hubgit/e394e9be07d95cd5e774989178139ae8?permalink_comment_id=3366379#gistcomment-3366379
```

Comment:
does this file actually get used anywhere?

I'm trying to figure out the difference between this file:
[src/components/selectField.js](https://github.com/codefordc/us-congress-pizza-flag-tracker-frontend/pull/144/files#diff-f55f82a2a4c9bc2fc1f52d8143f811a6945c33446ca8e43092922b2567ce49d2)

and this one:
[src/components/scanOrder/SelectField.js](https://github.com/codefordc/us-congress-pizza-flag-tracker-frontend/pull/144/files#diff-b1a645be2b714f691a83cec92d7f70071286880d21aae47d76a7d8a4e5609c97)